### PR TITLE
Create article form

### PR DESCRIPTION
**Pull request by alghazu**
### What we did:
- Create an edit view for articles

### CODE REVIEW

**Review by GeorgeJLewis — APPROVED** [review]
Looks good.

**Comment by GeorgeJLewis** [comment]
Fails in semaphore
This could be due to the merge conflicts?
```Created database 'b7eba302-3d79-4ba2-ab82-798311e5f050_development'
Created database 'b7eba302-3d79-4ba2-ab82-798311e5f050_test'
rake aborted!
ActiveRecord::DuplicateMigrationNameError: 

Multiple migrations have the name CreateArticles.```

**Comment by alghazu** [comment]
Fixed

**Review by GeorgeJLewis — APPROVED** [review]

**Comment by GeorgeJLewis** [comment]
LGTM

File: app/controllers/articles_controller.rb [hunk 1]
```diff
@@ -1,7 +1,25 @@
...
+    @article = Article.new
+  end
+
+  def create
+    @article = Article.new(article_params)
+    if @article.save
+      flash[:notice] = "Article successfully saved and sent for approval"
+      redirect_to root_path
+    else
+      render 'new'
```
Comment (magnus-thor) [inline app/controllers/articles_controller.rb]: Missing an error message if somethings goes wrong

**Review by magnus-thor — CHANGES_REQUESTED** [review]

File: app/controllers/articles_controller.rb (hunk 1 above)
Comment (magnus-thor) [reply app/controllers/articles_controller.rb]: Also you need to write a sad path if the user enters a empty field, you have validations in place that content or title cant be blank.

Comment (alghazu) [reply app/controllers/articles_controller.rb]: Done

**Review by magnus-thor — APPROVED** [review]

**Comment by magnus-thor** [comment]
Commit history is one sided, need to practice more git pong. Otherwise good job.

**Pull request merged**